Sources/GalleryShareButton.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  GalleryShareButton
+//  LegacyGallery
+//
+//  Created by Alexander Chekel on 01.02.2022.
+//
+
+import UIKit
+
+internal class GalleryShareButton: UIButton {
+    var isLoading: Bool {
+        get {
+            activityIndicatorView.isAnimating
+        }
+        set {
+            titleLabel?.isHidden = newValue
+
+            if newValue {
+                activityIndicatorView.startAnimating()
+            } else {
+                activityIndicatorView.stopAnimating()
+            }
+        }
+    }
+
+    override var contentEdgeInsets: UIEdgeInsets {
+        didSet {
+            let isRTL = semanticContentAttribute == .forceRightToLeft
```

Comment:
I'm not sure if this is the best way to check wether view is in RTL mode. Maybe it would be more appropriate to check it like `effectiveUserInterfaceLayoutDirection == .rightToLeft`.